include/ftl/atomic_counter.h

```diff
@@ -158,6 +158,30 @@ class AtomicCounter {
 		return prev;
 	}
 
+	/**
+	 * A wrapper over std::atomic_uint::compare_exchange_strong()
+	 *
+	 * The compare_exchange_strong *with* be atomic, but this function as a while is *not* atomic
+	 *
+	 * \param expectedValue The value that is expected to be in the atomic counter
+	 * \param newValue      The value that the atomic counter will be set to if comparison succeeds.
+	 * \param memoryOrder   The memory order to use for the compare_exchange_strong
```

Comment:
Small thing, but if you're in there fixing the above, can you have 4 spaces after the longest parameter value. Also, can you updated it to @param instead of \param. So the style matches the other comments

Reply:
:ok_hand: That's what my editor generates (as I use that style) but I obviously didn't catch them all.